<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info</class>
 <widget class="QWidget" name="Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Information</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background: #242A38;
	border: 2px solid white;
	border-radius: 2px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPlainTextEdit" name="text">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid none;
color: white;
font-size: 28em;
margin-right: 2em;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_title">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid none;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; color:#ffffff;&quot;&gt;TITLE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_no">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: #4E596F;
    border-width: 1px;
    border-radius: 13px;
    border-color: #4E596F;
    font: 12pt;
    color: white;
    padding: 10px;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 15px;
	margin-bottom: 10px;
	min-height: 20px;
}

QPushButton:hover {
background: qradialgradient(cx:0.3, cy:0.5, radius: 5.5,
              fx:0.5, fy:0.5, stop:0 #4E596F, stop:1 white);
}

QPushButton:pressed {
border-style: inset;
background: qradialgradient(cx:0.7, cy:0.5, radius: 5.5,
              fx:0.5, fy:0.5, stop:0 #4E596F, stop:1 black);
}

QPushButton:focus {
   border: none;
   outline: none;
}</string>
       </property>
       <property name="text">
        <string>SHARE LINK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_yes">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background: #f76163;
    border-width: 1px;
    border-radius: 13px;
    border-color: #f76163;
    font: 12pt;
    color: white;
    padding: 10px;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 15px;
	margin-bottom: 10px;
	min-height: 20px;
}

QPushButton:hover {
background: qradialgradient(cx:0.3, cy:0.5, radius: 5.5,
              fx:0.5, fy:0.5, stop:0 #f65964, stop:1 white);
}

QPushButton:pressed {
background: qradialgradient(cx:0.7, cy:0.5, radius: 5.5,
              fx:0.5, fy:0.5, stop:0 #f65964, stop:1 black);
}

QPushButton:focus {
   border: none;
   outline: none;
}</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
